src/Agent.Worker/AgentPluginManager.cs

```diff
@@ -32,8 +32,9 @@ public sealed class AgentPluginManager : AgentService, IAgentPluginManager
         {
             "Agent.Plugins.Repository.CheckoutTask, Agent.Plugins",
             "Agent.Plugins.Repository.CleanupTask, Agent.Plugins",
-            "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask, Agent.Plugins",
-            "Agent.Plugins.PipelineArtifact.PublishPipelineArtifactTask, Agent.Plugins"
+            "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTaskV0, Agent.Plugins",
```

Comment:
this will break compat with old task, leave `"Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask, Agent.Plugins",` as it is.